pacu/modules/iam__enum_roles/main.py

```diff
@@ -159,7 +160,7 @@ def main(args, pacu_main):
     iam: 'mypy_boto3_iam.IAMClient' = pacu_main.get_boto3_client('iam')
 
     if args.role_name:
-        role_name = args.role_name
+        role_name = args.role_name.split('/')[-1]  # Handle ARN's if that was passed for whatever reason.
```

Comment:
Works the same as before but handles cases where a role ARN was passed instead of a name.